src/components/test/common-tests.js

```diff
@@ -334,12 +339,12 @@ export function testTransitionComponent(
           onTransitionEnd,
         });
 
-        // Default to the main container as the `target` for the TransitionEvent,
-        // as that would be the actual behavior at runtime.
-        // If that's not desired, callers can still override it by providing
-        // event.target explicitly.
-        const container = wrapper.find('div');
+        const container = wrapper.find(elementSelector);
         container.prop('ontransitionend')({
+          // Default to the main container as the `target` for the TransitionEvent,
+          // as that would be the actual behavior at runtime.
+          // If that's not desired, callers can still override it by providing
+          // event.target explicitly.
```

Comment:
This comment was misplaced. It references the setting of the `target` prop, so I have moved it down here.